fractal-puppy/Engine
# Bug/level2 weapon mesh bug

**WARNING: PR #316 must be merged before this one.**

**Before:** If you had a weapon equipped on level 1, when you entered level 2 it was still equipped but the mesh didn't appear.
**Now:** The weapon mesh appears as well.

**To test:**
1. Open the app and play the game from the first level.
2. Pick up the weapon on the floor and equip it.
3. Reach level 2.
4. See if the weapon is still equipped.
